addons/stock/models/product.py

```diff
@@ -514,13 +514,7 @@ def get_theoretical_quantity(self, product_id, location_id, lot_id=None, package
     def write(self, values):
         res = super(Product, self).write(values)
         if 'active' in values and not values['active']:
-            products = self.mapped('orderpoint_ids').filtered(lambda r: r.active).mapped('product_id')
-            if products:
-                msg = _('You still have some active reordering rules on this product. Please archive or delete them first.')
-                msg += '\n\n'
-                for product in products:
-                    msg += '- %s \n' % product.display_name
-                raise UserError(msg)
+            self.orderpoint_ids.active = False
```

Comment:
@oma-odoo If I archive the product and active it again, I won't reactivate the linked orderpoints also?